package.json

```diff
@@ -39,59 +39,59 @@
-    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
-    "@babel/plugin-syntax-import-meta": "^7.0.0",
-    "@babel/plugin-transform-runtime": "^7.1.0",
-    "@babel/preset-env": "^7.1.5",
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2",
+    "@babel/plugin-proposal-class-properties": "^7.1.2",
+    "@babel/plugin-proposal-do-expressions": "^7.1.2",
+    "@babel/plugin-proposal-export-default-from": "^7.1.2",
+    "@babel/plugin-proposal-export-namespace-from": "^7.1.2",
+    "@babel/plugin-proposal-json-strings": "^7.1.2",
+    "@babel/plugin-proposal-logical-assignment-operators": "^7.1.2",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.1.2",
+    "@babel/plugin-proposal-numeric-separator": "^7.1.2",
+    "@babel/plugin-proposal-optional-chaining": "^7.1.2",
+    "@babel/plugin-proposal-pipeline-operator": "^7.1.2",
+    "@babel/plugin-proposal-throw-expressions": "^7.1.2",
+    "@babel/plugin-syntax-dynamic-import": "^7.1.2",
+    "@babel/plugin-syntax-import-meta": "^7.1.2",
+    "@babel/plugin-transform-runtime": "^7.1.2",
+    "@babel/preset-env": "^7.1.2",
     "@babel/preset-flow": "^7.0.0",
-    "@babel/runtime": "^7.1.5",
+    "@babel/runtime": "^7.1.2",
     "@integreat-app/eslint-config-babel": "^0.3.1",
     "@integreat-app/eslint-config-flow": "^0.3.2",
     "@integreat-app/eslint-config-jest": "^0.2.1",
     "@integreat-app/eslint-config-node": "^0.3.2",
-    "babel-core": "^6.26.3",
     "babel-eslint": "^10.0.1",
-    "babel-jest": "^23.6.0",
-    "babel-loader": "^8.0.4",
+    "babel-jest": "^24.0.0",
+    "babel-loader": "^8.0.5",
     "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-preset-env": "^1.7.0",
-    "es-check": "^4.0.0",
-    "eslint": "^5.9.0",
+    "es-check": "^5.0.0",
+    "eslint": "^5.12.1",
     "eslint-config-standard": "^12.0.0",
-    "eslint-plugin-flowtype": "^3.2.0",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jest": "^22.0.0",
-    "eslint-plugin-node": "^8.0.0",
+    "eslint-plugin-flowtype": "^3.2.1",
+    "eslint-plugin-import": "^2.15.0",
+    "eslint-plugin-jest": "^22.1.3",
+    "eslint-plugin-node": "^8.0.1",
     "eslint-plugin-prefer-object-spread": "^1.2.1",
     "eslint-plugin-promise": "^4.0.1",
     "eslint-plugin-standard": "^4.0.0",
-    "flow-bin": "^0.86.0",
+    "flow-bin": "^0.91.0",
     "flow-copy-source": "^2.0.2",
     "flow-typed": "^2.5.1",
-    "jest": "^23.6.0",
-    "jest-fetch-mock": "^1.7.4",
-    "jest-junit": "^5.2.0"
+    "jest": "^24.0.0",
+    "jest-fetch-mock": "^2.1.0",
+    "jest-junit": "^6.1.0",
+    "shelljs": "^0.8.3",
+    "webpack": "^4.29.0",
+    "webpack-cli": "^3.2.1"
   },
   "dependencies": {
     "entities": "^1.1.2",
     "is-url": "^1.2.4",
-    "lodash-es": "^4.17.11",
-    "moment": "^2.22.2",
+    "lodash": "^4.17.11",
+    "moment": "^2.24.0",
     "sanitize-html-react": "^1.13.0",
-    "webpack": "^4.25.1",
-    "webpack-cli": "^3.1.2"
+    "webpack-node-externals": "^1.7.2"
   }
```

Comment:
I think these dependencies should have less strict version-numbers.
```
"entities": "1.x.x",
[...]
```
should be enough I guess (and better for dependents on this lib)

Reply:
` ^1.2.3 will match any 1.x.x release including 1.3.0, but will hold off on 2.0.0.`

Reply:
No, it will only allow 1.2.3 upwards on minor releases. It won't allow versions 1.2.2 or below.

Reply:
I'd prefer 1.x.x so libs don't (unnecessarily) force dependents to use new minor versions...

Reply:
To what exactly do you want to change that line? 
`"entities": "1.x.x",` is not valid afaik

Reply:
Yup it is: https://docs.npmjs.com/misc/semver#x-ranges-12x-1x-12-

Reply:
You could also just do 
```
"entities": "1"
```

Reply:
k, still dont get it why we need it in this case. In order to use 1.x.x I actually have to go though every version and check if that version works with out project.

Else we won't get warnings that we a version conflict.

Reply:
Actuelly I would have to do this for the other libs too... technically

Reply:
Well it's only a suggestion^^ we dont *need* to it like this.

*TLDR*: New suggestion: Use `^1.2`.

My reasoning for it is:
Once a dependent (of this lib) needs "entities", but in version "1.1.1", because there was some misbehavior introduced in "1.1.2", then this would collide with our lib if we set entities to "^1.1.2".
The downside is, as you said, that we haven't tested our lib against all minor version of below "1.1.x". ~On the other hand with "^1.2.3" you also assume that any upcoming minor version will work with our lib. But then we can also trust these creators to already have had a stable API throughout "1.0.0" until "1.2.3".~

According to semver, new minor versions are backwards compatible while adding new functionalities and new patch versions are only used for bug-fixes (and new major versions for breaking changes of course).

I changed my mind to the following: If we don't want to test our lib to work with all minor versions <1.2.x, we shouldn't set the version range to "1.x.x" (because 1.2.0 could have introduced some new functionality that we depend on), but what we can do is set it to "^1.2" instead.
 So we can specify that our lib depends on the API of major version 1 and minor version 2 (which allows any minor version >= 2 according to semver)  --- while ignoring the actual patch version: that's up to the dependent of our lib whether he uses the latest patches of "entities" or not